.travis.yml

```diff
@@ -13,8 +13,7 @@ install:
   - python -m pip install -U pip wheel setuptools
   - python -m pip install -U isort flake8
   - python -m pip install -r requirements.txt
-  - python -Wd setup.py sdist
-  - python -Wd -m pip install dist/py3-validate-email-*.tar.gz
+  - python -Wd setup.py develop  # needed to download blacklist in source dir
```

Comment:
Nope!

The point here is to install the package as if it were installed from pypi. `setup.py develop` is not the way of doing it!

Reply:
Okay, understood!